packages/commerce-sdk-react/src/hooks/ShopperBaskets/action.ts

```diff
@@ -328,16 +328,16 @@ the body are the following properties if specified:
      * @see {@link https://developer.salesforce.com/docs/commerce/commerce-api/references/shopper-baskets?meta=addTaxesForBasket} for more information about the API endpoint.
      * @see {@link https://salesforcecommercecloud.github.io/commerce-sdk-isomorphic/classes/shopperbaskets.shopperbaskets-1.html#addtaxesforbasket} for more information on the parameters and returned data type.
      */
-    AddTaxesForBasket = 'addTaxesForBasket'
+    AddTaxesForBasket = 'addTaxesForBasket',
```

Comment:
is this intended? Are we having a trailing comma now?

Reply:
fixed